MerkleTreeAirdrop/README.md

```diff
@@ -0,0 +1,49 @@
+# MerkleTreeAirdrop
+
+## Overview
+
+Requires a list of wallet address and root hash which are the leaves and merkle proofs needed to verify that a given data exists for that address in the tree.
+Contract acts as Airdrop and only sends out tokens on data verification i.e merkle proof and appropriate arguments used to build the merkle tree for that user.
+
+## How to use
+
+1. Prepare airdrop token.
```

Comment:
The airdrop token should be ERC20 token, not ERC721 and ERC1155.
You may change it "Prepare an ERC20 token as an airdrop token. You can use existing token or deploy a new ERC20 token on DApps page"